aws/required-permissions.md

```diff
@@ -129,3 +131,82 @@ Check out the templates directly for more information.
 | `s3:PutObjectTagging` | Action | Put tags on the scanned object. |
 | `sqs:SendMessage` | Action | Send scan messages to the `ScannerQueue`. |
 | [AWSLambdaVPCAccessExecutionRole](https://console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole$jsonEditor?) | Managed Policy | **Optional.** Provides minimum permissions for a Lambda function to execute while accessing a resource within a VPC - create, describe, delete network interfaces and write permissions to CloudWatch Logs. |
+
+## FSS Account Scanner Stack
+
+### Permissions required by account scanner stack
+
+> **Note:** The Permissions for ManagementRole used by Cloud One File Storage Security to maintain the account scanner stack are put into next section.
+
+| Permission | Type | Description |
+| --- | --- | --- |
+| [AWSLambdaBasicExecutionRole](https://console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole$jsonEditor) | Managed Policy | Provides write permissions for a Lambda function to CloudWatch Logs. |
```

Comment:
```suggestion
| [AWSLambdaBasicExecutionRole](https://console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole$jsonEditor) | Managed Policy | Provides write permissions for a Lambda function to CloudWatch Logs |
```
The third column contains a clause phrase, not a full sentence, so doesn't require a period.